packages/ui/packages/InputRange/src/index.js

```diff
@@ -32,7 +33,7 @@ const InputRange = props => {
           onChange={setValue}
           onChangeComplete={onChangeComplete}
           formatLabel={() => false}
-          rtl
+          rtl //@tood: it should be an optional prop, use might want to use LTR
```

Comment:
this will not listed as `todo`

Reply:
any improvement would be accepted since this is an improvement and the entire UI components can work LTR, this should be also!

Reply:
it is ok, but `tood` is not `todo` :) I meant to fix that typo to listed as todos in editors

Reply:
bro, not just fixing the todo 'word' , but i implemented the thing i wanted to :))